packages/hint-no-disallowed-headers/src/_locales/en/messages.json

```diff
@@ -11,6 +11,30 @@
         "description": "Report message when the 'server' header contains more than the server name",
         "message": "The 'server' header should only contain the server name."
     },
+    "disallowedExpiresHeader": {
+        "description": "Report message when the response includes the Expires header",
+        "message": "The 'Expires' header should not be used, 'Cache-Control' should be preferred."
+    },
+    "disallowedHostHeader": {
+        "description": "Report message when the response includes the Host header",
+        "message": "The 'Host' header should not be used, it is a request header only."
+    },
+    "disallowedP3PHeader": {
+        "description": "Report message when the response includes the P3P header",
+        "message": "The 'P3P' header should not be used, it is a non-standard header only implemented in Internet Explorer."
+    },
+    "disallowedPragmaHeader": {
+        "description": "Report message when the response includes the Pragma header",
+        "message": "The 'Pragma' header should not be used, it is deprecated and is a request header only."
+    },
+    "disallowedViaHeader": {
+        "description": "Report message when the response includes the Via header",
+        "message": "The 'Via' header should not be used, it is a request header only."
+    },
+    "disallowedXFrameOptionsHeader": {
+        "description": "Report message when the response includes the X-Frame-Options header",
+        "message": "The 'X-Frame-Options' header should not be used, the same effect can be achieved with more consistent support using the 'Content-Security-Policy' header."
```

Comment:
I suggest explicitly mentioning the [`frame-ancestors`](https://www.w3.org/TR/CSP/#directive-frame-ancestors) navigation directive here as it might save developers some time if they're unfamiliar with it.

(Also, while I'm not sure there's much benefit in describing it here, `frame-ancestors` and `X-Frame-Options` doesn't have the _exact_ same effect, `frame-ancestors` is stronger in that it checks against _any_ ancestors where `X-Frame-Options` only checks the top-level document's location as described here: https://www.w3.org/TR/CSP/#frame-ancestors-and-frame-options)

Reply:
Thanks, it makes sense to me to mention the `frame-ancestors` directive. I'll update the message.